EnglishFilter/sections/specific.txt

```diff
@@ -4110,7 +4109,7 @@ cloudstorageoptions.com##.adslot_left
 ||adee.hit.gemius.pl/gdejs/xgde.js$domain=sportacentrs.com
 ||webgate.tn/getlink/*.js
 ||webgate.tn/promo/*.gif
-techsapparel.com,urlty.in,semawur.com,idzone.site##center > a[href] > img
+tokyoghoulre.com,watchgoblinslayer.com,watchnoblesse.com,watchsnk.net,watchtowerofgod.com,watchdrstone.com,demonslayeranime.com,watchblackclover.com,watchkaguyasama.com,haikyuu3.com,watchoverlord2.com,watchboruto.com,watchsao.tv,watchrezero2.com,getpronews.com,techsapparel.com,urlty.in,semawur.com,idzone.site##center > a[href] > img
```

Comment:
Remove `getpronews.com` from the rule.

Reply:
@zloyden Actually `abort-current-inline-script` breaks the whole `String.fromCharCode` function, as seen in the Console Log. It doesn't happen when I use uBO's `acis` in uBO.